pcdsdevices/slits.py

```diff
@@ -643,3 +643,18 @@ class SimLusiSlits(LusiSlits):
     ywidth = Cpt(FastMotor)
     xcenter = Cpt(FastMotor)
     ycenter = Cpt(FastMotor)
+
```

Comment:
There's a style issue here (two blank lines between class definitions) that will cause continuous integration checks of your pull request to fail.

Did you configure pre-commit? It would have caught this issue before even letting you commit the changes. Usage is documented briefly here:
https://confluence.slac.stanford.edu/display/PCDS/How+to+add+a+device+to+pcdsdevices#Howtoaddadevicetopcdsdevices-Pre-commit

Alternatively, see the `flake8` section above that.